app/scripts/directives/showGroup.js

```diff
@@ -0,0 +1,55 @@
+import template from 'views/components/showGroup.html';
+
+angular.module('confRegistrationWebApp')
+  .component('showGroup', {
```

Comment:
I did kinda like having modal in the component name and filename. Makes it clear what it is. `showGroupModal` is good.